misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -337,15 +336,34 @@ class JettyService @Inject internal constructor(
         // the address is a null byte ('\0'). The address has no connection with filesystem
         // path names.
       }
-      server.destroy()
+
+      logger.info { "Stopped Jetty in $stopwatch" }
     }
 
     if (healthExecutor != null) {
```

Comment:
this should be fine to be called twice, but we don't really care about errors that late into shutdown anyway -- i.e. i can always add the check later